src/SonarScanner.MSBuild.PreProcessor/JreResolution/JreCache.cs

```diff
@@ -105,6 +105,7 @@ public async Task<JreCacheResult> DownloadJreAsync(string sonarUserHome, JreDesc
 
     private async Task<Exception> DownloadAndValidateJre(string jreDownloadPath, string downloadTarget, string sha256, Func<Task<Stream>> jreDownload)
     {
+        logger.LogWarning(Resources.WARN_JreDownloadBottleneck);
```

Comment:
I chose warning, would you prefer Info?

Reply:
Info should be enough. We actually don't want users to fix the version, which is what `skipJreProvisioning` is actually doing.

Reply:
Move the logging before ` await jreDownload();` because file creation may fail before the actual download.

Reply:
I would also mention the `jreDescription.Filename` in the message, so the users know what we were about to install as a compatible JRE.